langkit/templates/pkg_rewriting_impl_body_ada.mako

```diff
@@ -27,11 +27,138 @@ package body ${ada_lib_name}.Rewriting_Implementation is
      (Rewriting_Handle_Pointer, Rewriting_Handle);
    pragma Warnings (On, "possible aliasing problem for type");
 
+   --------------
+   -- PRE_FAIL --
+   --------------
```

Comment:
Please use our standard coding convention: case-preserving comment boxes, one comment box per subprogram and the documentation with the subprogram declaration (not body, so add declarations for all subprogram bodies). Thanks!

Reply:
The commit right after this one removes these subprograms… so at this point, why not merging the two commits to do the right thing from the start?